libraries/azurerm_ad_user.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+
+class AzurermAdUser < AzurermResource
+  name 'azurerm_ad_user'
+  desc 'Verifies settings for an Azure Active Directory User'
+  example <<-EXAMPLE
+    describe azure_ad_user(azureId: 'azure_id') do
```

Comment:
This doesn't match the `initialize` method arguments. Suggest either removing the declaration and just passing in the id or changing the method to match.

Also, would `user_id` be a better choice?

Reply:
In the response on the plural resource the field is called `ObjectId`.